NSS-Day-Cohort-42/holiday-road-eccentric-eels
# Added Weather component to display current weather data for specified park location.

# Steps to Review

1. `git fetch --all`
1. `git checkout mp-weather`
1. make sure your json server is running
1. make sure your api keys are entered into the Settings.js file
1. serve to browser and verify that weather data populates after selecting a park
1. review code in the weather folder to confirm coding logic is sound.
